src/plugins/multitrack-media/multitrack-media.yml

```diff
@@ -1,3 +1,6 @@
+name:
+  en: Multitrack media
+  de: Mehrspur Medien
```

Comment:
Better would be to have a hyphen between the words, I guess

Reply:
👍 Added